app/css/vendors/ip-camera.css

```diff
@@ -0,0 +1,77 @@
+.camera-view {
+  display: flex;
+  flex-direction: column;
+}
+
+.camera-view__body {
```

Comment:
__question:__ Do you have already any guidelines for the CSS naming/style? I was trying something BEM-like to have less nested selectors. What do you think?